example/models/food/v1/food.pb.pgdb.go

```diff
@@ -1692,6 +1786,32 @@ func (x *PastaIngredientDBQueryBuilder) FTSData() *PastaIngredientFTSDataSafeOpe
 	return &PastaIngredientFTSDataSafeOperators{tableName: x.tableName, column: "pb$" + "fts_data"}
 }
 
+type PastaIngredientModelEmbeddings_MODEL_GPT_3_5_XXXSafeOperators struct {
+	column    string
+	tableName string
+}
+
+func (x *PastaIngredientModelEmbeddings_MODEL_GPT_3_5_XXXSafeOperators) Identifier() exp.IdentifierExpression {
+	return exp.NewIdentifierExpression("", x.tableName, x.column)
+}
+
+func (x *PastaIngredientDBQueryBuilder) ModelEmbeddings_MODEL_GPT_3_5_XXX() *PastaIngredientModelEmbeddings_MODEL_GPT_3_5_XXXSafeOperators {
+	return &PastaIngredientModelEmbeddings_MODEL_GPT_3_5_XXXSafeOperators{tableName: x.tableName, column: "pb$" + "model_embeddings_1"}
+}
+
+type PastaIngredientModelEmbeddings_MODEL_GPT_4_0_XXXSafeOperators struct {
```

Comment:
@ButternCream doesnt seem like the TrimPrefix is working as we think?

Reply:
https://github.com/ductone/protoc-gen-pgdb/pull/66/commits/abc3f6dd4671b5ff85eea12658f43c9faca52f53